subworkflows/nf-core/bam_trim_primers_ivar/main.nf

```diff
@@ -0,0 +1,44 @@
+include { IVAR_TRIM               } from '../../../modules/nf-core/ivar/trim/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow BAM_TRIM_PRIMERS_IVAR {
+    take:
+    bam   // channel: [ val(meta), [ bam ], [bai] ]
+    bed   // path   : bed
+    fasta // channel: reference.fasta
```

Comment:
```suggestion
    ch_bam   // channel: [ val(meta), path(bam), path(bai) ]
    ch_bed   // channel: [ path(bed) ]
    ch_fasta // channel: [ path(fasta) ]
```

Can you also change the output comments to this format? All channels should start with `ch_` (except for the output names of the subwf) and all values should start with `val_`